connectors/src/connectors/github/index.ts

```diff
@@ -263,7 +272,7 @@ export class GithubConnectorManager extends BaseConnectorManager<null> {
         nodes = nodes.concat(
           page.map((repo) => ({
             provider: c.type,
-            internalId: repo.id.toString(),
+            internalId: getRepositoryNodeId(repo.id),
```

Comment:
In connector I think we should solely rely on the terminology "internalId" and refrain from using the terminology "nodeId"

Reply:
done